pkg/raft/rafttest/interaction_env_handler.go

```diff
@@ -204,6 +204,14 @@ func (env *InteractionEnv) Handle(t *testing.T, d datadriven.TestData) string {
 		}
 		env.Output.WriteString(err.Error())
 	}
+
+	// Check invariants on every node.
+	for _, n := range env.Nodes {
+		if err := n.CheckInvariants(); err != nil {
```

Comment:
Don't know if this is a good approach. Perhaps the invariants should better be checked exhaustively in smaller unit-tests. But this catch-all is a convenient place for quick checks.